### PR TITLE
Update Microsoft.Azure.WebJobs and Microsoft.Azure.WebJobs.Logging.ApplicationInsights to v3.0.25

### LINKED ISSUE #2548: Add ability to turn on SQL query logging for Azure Functions
Before 28 july 2020 we were able to see the sql queries in appinsight dependencies logs, but now we are only seeing server names and no sql queries are logged from azure functions consumption plan.  We have other code in aks cluster and sql queries are logged for calls done for app hosted in aks.

#### Expected behavior

Sql queries not logged in appinsight for azure function consumption plan.

#### Actual behavior

Sql queries should be logged in appinsight for azure function consumption plan.

#### Known workarounds
i tried this link but it didnt help
https://github.com/microsoft/ApplicationInsights-dotnet/issues/1425
Also adding reference to this dll brings aspnetcore things in azurefuctions project and is not making sense 

#### Related information 

Provide any related information 

* Package version
* Links to source
image

### PR DESCRIPTION
**Pull request by soninaren**
### Issue describing the changes in this PR
Fetching ApplicationInsights and related packages that expose dependency tracking options

resolves https://github.com/Azure/azure-webjobs-sdk/issues/2548

### Pull request checklist

* [x] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [x] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

File: src/WebJobs.Script/WebJobs.Script.csproj [hunk 1]
```diff
@@ -44,10 +44,10 @@
     <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="2.0.6" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS6" Version="3.0.557" />
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker.PS7" Version="3.0.560" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.23" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.25" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="4.0.1" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.8" />
```
Comment (yojagad) [inline src/WebJobs.Script/WebJobs.Script.csproj]: Was this going to be updated as well ? Microsoft.Azure.WebJobs.Extensions.Http

**Review by yojagad — DISMISSED** [review]

File: src/WebJobs.Script/WebJobs.Script.csproj (hunk 1 above)
Comment (soninaren) [reply src/WebJobs.Script/WebJobs.Script.csproj]: done

File: release_notes.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 -->
 - Fixed [bug](https://github.com/Azure/azure-functions-durable-extension/issues/1467) in sync triggers operations for Durable Functions using custom storage account connection strings.
 - Update PowerShell Worker to 3.0.557 (PS6) [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.557) and 3.0.560 (PS7) [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.560)
+- Update Microsoft.Azure.WebJobs and Microsoft.Azure.WebJobs.Logging.ApplicationInsights to v3.0.25 [Release Notes](https://github.com/Azure/azure-webjobs-sdk/releases/tag/v3.0.25)
```
Comment (alrod) [inline release_notes.md:7]: Should we add a note that we are updating http ext?

**Review by mathewc — DISMISSED** [review]

File: release_notes.md (hunk 2 above)
Comment (soninaren) [reply release_notes.md:7]: yeah let me add the note

**Review by alrod — DISMISSED** [review]

**Review by mathewc — DISMISSED** [review]

**Review by mathewc — APPROVED** [review]

**Pull request merged**